README.md

```diff
@@ -1,13 +1,15 @@
-# WebAssembly Reference Manual README.md
+# WebAssembly Reference Manual
 
-This directory contains
-[WebAssembly.md](https://github.com/sunfishcode/wasm-reference-manual/tree/master/WebAssembly.md),
-a human-readable reference manual document written in Github-Flavored Markdown.
+This project contains a [human-readable WebAssembly reference manual document written in Markdown](https://github.com/sunfishcode/wasm-reference-manual/tree/master/WebAssembly.md). It is intended as an alternative to the [official WebAssembly specification](https://webassembly.github.io/spec/), which is not terribly accessible or readable. If you want to know how the WebAssembly runtime works, are writing a compiler, or want to try writing WebAssembly by hand (in text format), this manual is a good starting point.
```

Comment:
I appreciate what you're saying here, though I'd like to be a little more nuanced in the phrasing.

> which is not terribly accessible or readable

A charitable interpretation of the official spec is that it's written for a specific target audience. How about we say something like "which is organized around a particular approach to language specification, rather than on explanation"?

Reply:
That's a good point, I was being a little unfair - as a WebAssembly user, I guess i am not the target audience of the official spec.

I've updated the text.